### PR TITLE
completed js-primitives-and-variables

### PR DESCRIPTION
**Pull request by HuynhCynthia**
Screen Shot 2023-01-30 at 10 46 32 AM

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const fullName = 'Cynthia Huynh';
+var isCool = false;
+const totalPets = 1;
+const unknown = null;
+
```
Comment (shawnkost) [inline javascript-primitives-and-variables/main.js:5]: I like that you're already using `const`, but you should stick to using `var` for now.

File: javascript-primitives-and-variables/main.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+console.log('Full Name:', fullName);
+console.log('Is Cynthia Cool?:', isCool);
+console.log('How may pets does she have? If less than 5, that is why not cool:', totalPets);
+console.log(typeof fullName);
+console.log(typeof isCool);
+console.log(typeof totalPets);
+console.log('haha you suck:', unknown);
+console.log(typeof unknown);
```
Comment (shawnkost) [inline javascript-primitives-and-variables/main.js:13]: You should have one more variable `never` with a couple of console logs tied to it.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Cynthia! A couple changes requested below. You're missing a variable, and let's stick to using `var` for now until you start to learn about ES6

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**